reports-backend/src/main/java/org/jboss/da/listings/api/model/Product.java

```diff
@@ -0,0 +1,62 @@
+package org.jboss.da.listings.api.model;
+
+import org.jboss.da.listings.api.service.ArtifactService.SupportStatus;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Set;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * 
+ * @author Jozef Mrazek <jmrazek@redhat.com>
+ *
+ */
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@ToString
+@Entity
+public class Product extends GenericEntity {
+
+    @Setter
+    @Getter
+    @NonNull
+    private String name;
+
+    @Setter
+    @Getter
+    @NonNull
+    private String productVersion;
```

Comment:
I think it doesn't respect the created design: The productVersion and support status  (+ id) should be in the separated entity ProductVersion.